src/main.ts

```diff
@@ -362,6 +363,13 @@ export default class LinterPlugin extends Plugin {
     } catch (error) {
       this.handleLintError(file, error, 'Lint File Error in File \'${file.path}\'', false);
     }
+
+    oldText = editor.getValue();
+    newText = this.rulesRunner.runCustomRegexReplacement(this.settings.customRegexs, oldText);
+    const lines = oldText.split('\n');
+    const start = {line: 0, ch: 0};
+    const end = {line: lines.length - 1, ch: lines[lines.length - 1].length};
+    editor.replaceRange(newText, start, end, oldText);
```

Comment:
I am not sure this needs to be its own thing. We should be able to call `runCustomRegexReplacement` from the `lintText` method and not have to worry about recreating the logic used for running these rules for a file.